frontend/src/components/adjudications/AdjudicateForm.jsx

```diff
@@ -46,8 +47,13 @@ export default function AdjudicateForm(props) {
             .then(({data}) => {
                 setAdjudications(data)  
                 setLoading(false)
-            });
-    }, []); //added the empty array so that it will only be called after the component mounts
+            });  
+
+        getNextUnjudgedPerformance(eventId, 1) //hardcoded tablet id for now
+            .then(({data}) => {
+                setNextPerformance(data)
```

Comment:
would this allow the adjudicator to press previous and return? or would this automatically redirect them?